### DISCUSSION TITLE
How to integrate the draw.io in React application.

### DISCUSSION THREAD

**Discussion by vamsilearner** [jgraph/drawio#3601] [Q&A]
How to integrate the draw.io in React application.

**Comment by chandaniKhanesha59** [comment]
import { DrawIoEmbed, DrawIoEmbedRef } from 'react-drawio';

 
  <DrawIoEmbed
        ref={drawioRef}
        urlParameters={{
          // ui: 'min', //'min' | 'atlas' | 'kennedy' | 'dark'

          spin: false,
          libraries: true,
          saveAndExit: true,
        }}
        onExport={(data) => {
          console.log(data, 'onExport');
          if (drawioRef.current) {
            drawioRef.current.draft({
              xml: data.xml,
              name: 'Helllllo',
              editKey: '',
              discardKey: '',
              ignore: false,
            });

            console.log(img, 'img');
          }
          setxml(data.xml);
          setimg(data.data);
        }}
        // onExport={(data) => console.log(data, 'onExport')}
        onLoad={(data) => {
          console.log(data, 'data');
        }}
        onSave={(data) => {
          console.log(data, 'onSave');
        }}
      />
